app/Http/Controllers/Table/PeeringDetailsController.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * PeeringDetailsController.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @link       https://www.librenms.org
+ *
+ * @copyright  2018 Tony Murray
+ * @author     Tony Murray <murraytony@gmail.com>
```

Comment:
Author and copyright should be fixed

Reply:
I figured Tony should retain the copyright and authorship as I merely copied his code and changed two words.

Reply:
You can then put 2 authors and your copyright.

Reply:
There we go!